src/components/help-panel/non-desktop-shortcuts-tabs.spec.tsx

```diff
@@ -0,0 +1,116 @@
+import { screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { HelpPanelSelectedTab } from "../../types";
+import { renderWithTheme } from "../../utils/testing-utils/render-with-theme";
+import { NonDesktopShortcutTabs } from "./non-desktop-shortcuts-tabs";
+
+import { useAppLayout } from "../../utils/layout";
+
+jest.mock("../../../public/images/trackpadTabMobile.svg");
```

Comment:
I thought svgr icons are mocked globally. You probably don't need to mock them in tests.

Reply:
done